### PR TITLE
feat(...): extend rule schema for internal post-processor

### PR DESCRIPTION
**Pull request by yosefAlsuhaibani**
- [x] I ran `make setup && make` to update the generated code after editing a `.atd` file (TODO: have a CI check)

### CODE REVIEW

File: rule_schema_v1.yaml [hunk 1]
```diff
@@ -953,7 +957,6 @@ properties:
               patterns: false
               pattern: false
               pattern-either: false
-              pattern-regex: false
```
Comment (aryx) [inline rule_schema_v1.yaml:956]: Is this related to the post-processor change?

**Review by aryx — APPROVED** [review]

File: rule_schema_v1.yaml (hunk 1 above)
Comment (yosefAlsuhaibani) [reply rule_schema_v1.yaml]: not quite; restored!

**Comment by aryx** [comment]
BTW it's usually good after you make a PR to have yourself a quick look at the diff, to fix simple things like that.
You should be your first reviewer.
